### PR TITLE
Remove BOM

### PR DESCRIPTION
**Pull request by yamal-alm**
### :goal_net: What's the goal?
Same problem described in this PR https://github.com/Telefonica/mistica-android/pull/230

### :construction: How do we do it?
Do not use Compose Bom and use explicit version of each compose dependency. Fortunately it is the same for every of it. I used 1.6.6 version, the one corresponding to "2024.04.01" BOM (defined in this mappings table https://developer.android.com/develop/ui/compose/bom/bom-mapping)

### :blue_book: Documentation changes?
- [x] No docs to update nor create

### :test_tube: How can I test this?
- [x] Manually tested, everything works as before

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -60,10 +60,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.11.0'
 
-    implementation platform("androidx.compose:compose-bom:$compose_bom_version")
-    implementation "androidx.compose.ui:ui"
-    implementation "androidx.compose.material:material"
-    implementation "androidx.compose.ui:ui-tooling-preview"
+    //noinspection BomWithoutPlatform
```
Comment (jdelga) [inline app/build.gradle]: I think this is only needed if we using a Bom: https://googlesamples.github.io/android-custom-lint-rules/checks/BomWithoutPlatform.md.html

Comment (yamal-alm) [reply app/build.gradle]: yes 🤦 this is from a intermediate modification, I forgot to remove it before pushing, thanks!

Comment (yamal-alm) [reply app/build.gradle]: removed 886a08dad8981b61e2ca8cbaf7ac8c0b733c56f6

**Review by jdelga — APPROVED** [review]

**Review by DevPabloGarcia — APPROVED** [review]
Thanks :)

**Pull request merged**